### DISCUSSION TITLE
[FF7] FFNx v1.14.0 & v1.15.0 Softlock in Northern Crater

### DISCUSSION THREAD

**Discussion by redmagejoe** [julianxhokaxhiu/FFNx#569] [Show and tell]
While using FFNx v1.14.0, I encountered a softlock behavior that traps your party inside the Northern Cave at the end of the game.  Suspecting this to be related to a specific mod, I thoroughly tested with and without every mod, before turning every single mod off and reproducing this behavior.  I then compared this to a vanilla version of the game through Steam without the use of 7th Heaven to confirm that it is indeed tied exclusively to the use of FFNx.

In the second screen of the Northern Cave, there are a number of ledges to jump down to descend further into the dungeon, some of which can be climbed up again to exit the dungeon.  On FFNx, one particular ledge behaves very peculiarly:  Upon attempting to climb up, no matter how far the player runs or what controls the player inputs, the character will turn towards the ledge, jump down a level, and then be stuck in a "no animation" state during any subsequent downward jumps.  At this point, the player has two choices:  Hop down to the next screen to "reset" this buggy state, or attempt to climb up again.  Choosing the latter will result in the character turning towards the ledge, staring at it for a moment, walking on "auto-pilot" towards a wall, and never giving control back to the player, soft-locking the game.

EDIT: I have only today updated to v1.15.0 and noticed that I still encounter this issue, but the parameters may now be more narrow than simply using FFNx at all. Testing ongoing in comments.

1. Launch the game without any mods, using the latest version of FFNx.
2. Save on the Highwind right before the final descent into the Northern Cave.
3. Move through the spiral ramp screen down to the second screen full of "hops" downward.
4. Attempt climb back up the ledge indicated in the screenshot, the ONLY means by which to exit the Northern Cave.
5. Witness strange "auto-pilot" behavior.
6. Attempt to climb up a second time, resulting in the softlock.

The expected behavior, as demonstrated on any version of the game not using FFNx, is for the player to be able to scale the ledge, retain control of their character, and leave the dungeon normally or move around as the player pleases.

![The offending ledge](https://i.imgur.com/5eH7mKW.jpeg)

**Comment by ChthonVII** [comment]
Does it happen in 13.0?
Does it happen with widescreen off?

**Comment by redmagejoe** [comment]
> Does it happen in 13.0? Does it happen with widescreen off?

Testing on v1.15.0, I have found that the problem does not occur when I set the Aspect Ratio to 4:3 and turn the Internal Resolution to 1x, and have 0 mods active.  The issue does not occur on 16:9 at 6x Internal Resolution with no mods, either.

EDIT:  I have narrowed down the issue. Compared to v1.14.0 where under any circumstances I would encounter this issue, on v1.15.0, the issue occurs while using only the 60/30 FPS Gameplay mod by Vertex2995 & Kaldarasha while on 16:9 and 6x Internal Resolution. I have not yet tested if the issue occurs on 4:3 with 1x Internal Resolution with the 60/30 FPS mod.  It is possible that the issue is exclusive to the 60/30 FPS mod on v1.15.0 of FFNx.

**Comment by redmagejoe** [comment]
This should likely be added to the list of 60 FPS bugs/glitches.

#347

**Reply by julianxhokaxhiu** [reply]
Yes please, consider adding this problem to the existing issue.
